packages/dataviews/src/types.ts

```diff
@@ -87,6 +87,12 @@ export type Field< Item > = {
 	 */
 	label?: string;
 
+	/**
+	 * The header of the field. Defaults to the label.
+	 * It allows the usage of a React Element to render the field labels.
+	 */
+	header?: string | ReactElement;
```

Comment:
Can we make this a ReactElement and remove the string type? Otherwise, why would you use the header as string if there's already the `label` field.

Unless you want to enable they being different, which I don't think is the intentionality here.

Reply:
It's just because in the `NormalizedField` type we fallback to the label to avoid having to do the fallback everywhere. But in `Field` it can indeed be just `ReactElement`

Reply:
To enforce the ReactElement type, when we normalize the field, can't we provide a function that returns the field.label if field.header is not present?